<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineReduction</class>
 <widget class="QWidget" name="LineReduction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="toolTip">
      <string>Number of lines the selected windows can have initially.</string>
     </property>
     <property name="text">
      <string>Maximal Window Lines:</string>
     </property>
     <property name="buddy">
      <cstring>max_window_lines</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="max_window_lines">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="value">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="toolTip">
      <string>When the truth table is not reversible, obtained by a window with maximal window lines, then the number of lines can be increased up at most this value.</string>
     </property>
     <property name="text">
      <string>Maximal Grow Up:</string>
     </property>
     <property name="buddy">
      <cstring>max_grow_up_window_lines</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="max_grow_up_window_lines">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="value">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="toolTip">
      <string>The possible window inputs are obtained by simulating its cone of influence. It is only simulated if the number of its primary inputs is less or equal to this value.</string>
     </property>
     <property name="text">
      <string>Window Variables Threshold:</string>
     </property>
     <property name="buddy">
      <cstring>window_variables_threshold</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="window_variables_threshold">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="value">
      <number>17</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Run-time:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="runtime">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
